doc/nrf/includes/external_flash_nrf91.txt

```diff
@@ -0,0 +1,11 @@
+To use the external flash memory, you must program the :ref:`nrf9160_ug_intro` by completing the following steps:
```

Comment:
perhaps also mention somewhere the minimum DK version that has external flash... i think v0.14.0

Reply:
Updated the "external_flash_nrf91.txt" with nRF9160 DK and the supported versions.